nursery/enumerate-files-in-dotnet.yml

```diff
@@ -0,0 +1,23 @@
+rule:
+  meta:
+    name: enumerate files in dotnet
```

Comment:
can we merge this and the other `enumerate files...` rules?
- enumerate files via kernel32 functions
- enumerate files via kernel32 functions

Reply:
awesome, thanks!